### PR TITLE
FAQ revamp

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Cleaned up the FAQ section and added section on the whitelisting (IPs and URL).

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [URL whitelisting for content security policy](https://www.notion.so/rudderstacks/URL-whitelisting-for-content-security-policy-3965f6ecdd6c4101a011c0867670c63b)

### CODE REVIEW

File: faqs.mdx [hunk 1]
```diff
@@ -1,91 +1,93 @@
...
 </div>
 
-## Installation and Setup
+## Installation and setup
 
-#### 1. How do I quickly get started with RudderStack?
+#### How do I quickly get started with RudderStack?
 
-To quickly get started, you can [**sign up for RudderStack**](https://app.rudderstack.com/). Here, you can configure your sources and destinations and start building your data pipelines in no time.
+To quickly get started, you can [sign up for RudderStack](https://app.rudderstack.com/). Here, you can configure your sources and destinations and start building your data pipelines in no time.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply faqs.mdx:20]: I think we shouldn't. That page essentially asks the reader to sign up and do the same thing :P

File: faqs.mdx [hunk 2]
```diff
@@ -95,42 +97,47 @@ CURL <DATA_PLANE_URL>/health
 
 A sample command would look something like:
 
-```c
+```bash
 CURL https://hosted.rudderlabs.com/health
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply faqs.mdx:101]: should be rudderlabs

File: faqs.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ishachopra0401) [reply faqs.mdx:20]: oops.. no I was thinking about how to configure a source and destination (quick start) after signing up.

**Review by ishachopra0401 — APPROVED** [review]

File: faqs.mdx [hunk 3]
```diff
@@ -1,91 +1,93 @@
...
 </div>
 
-## Installation and Setup
+## Installation and setup
 
-#### 1. How do I quickly get started with RudderStack?
+#### How do I quickly get started with RudderStack?
 
-To quickly get started, you can [**sign up for RudderStack**](https://app.rudderstack.com/). Here, you can configure your sources and destinations and start building your data pipelines in no time.
+To quickly get started, you can [sign up for RudderStack](https://app.rudderstack.com/). Here, you can configure your sources and destinations and start building your data pipelines in no time.
```
Comment (ameypv-rudder) [inline faqs.mdx:20]:
```suggestion
To quickly [get started](https://rudderstack.com/docs/get-started/), you can [sign up for RudderStack](https://app.rudderstack.com/). Here, you can configure your sources and destinations and start building your data pipelines in no time.
```

**Pull request merged**